### ISSUE TITLE
Cannot browse packages

### ISSUE THREAD

**Issue by hsbasu** [ubuntu-flutter-community/software#505]
With recent commits, Software has stopped showing categories in the explore page. The categories are available only in the search page.

https://user-images.githubusercontent.com/41947504/201454388-3a6d14a3-2011-4fbe-83ae-6ac859b4aadc.mp4

**Comment by Feichtmeier** [comment]
Yes this page is under design review. It was not performing good in the old layout so we changed it to this very simple grid view.
When you start searching the filters show up though. But we'll get to this page again but I don't know if it will be exactly like before

**Comment by Feichtmeier** [comment]
@anasereijo I assigned this to you, because we need good design for this. I hope you don't mind :P

**Comment by Feichtmeier** [comment]
Fixed with https://github.com/ubuntu-flutter-community/software/pull/525
